convox/rack
# Migrate to launch template

### What is the feature/fix?

AWS is deprecating the Launch Configuration, they recommend using the Launch Configuration. New instances types are not being added to launch config, only to launch templates.

### Does it has a breaking change?

No, all features of Launch Config are available on Launch Template.

### How to use/test it?

1. Install the RC on a rack
2. Deploy a new app and use it

You can also set SpotInstanceBid and check if the instances are being created.

### Checklist
